docs/src/main/asciidoc/getting-started-dev-services.adoc

```diff
@@ -0,0 +1,353 @@
+that your application changes broke the tests!
+
+To fix the tests, open `src/test/java/org/acme/GreetingResource.test`
+and replace
+
+[source, java]
+----
+             .body(is("Hello from RESTEasy Reactive"));
+----
+
+with
+
+[source, java]
+----
+             .body(containsString("Hello"));
+----
+
+This still validates the HTTP endpoint, but it's more flexible
+about the expected output.
+You should see in your terminal that the tests are now passing.
+
+== Adding persistence
+
+=== Creating a Panache Entity
+
+1. To add the persistence libraries, run
+
+[source, shell]
+----
+quarkus ext add hibernate-orm-panache jdbc-postgresql
+----
+
+The application will record the names of people it greets. Define a simple Hibernate Entity
+by creating a `Greeting.java` class. Add the following content:
+
+[source, java]
+----
+import io.quarkus.hibernate.orm.panache.PanacheEntity;
+import jakarta.persistence.Entity;
+
+@Entity
+public class Greeting extends PanacheEntity {
+String name;
+}
+----
+
+The entity makes use of xref:{doc-guides}hibernate-orm-panache.adoc[Panache], a layer on top of Hibernate ORM.
+ Extending `PanacheEntity` brings in a range of methods for reading, writing, and finding data.
+ Because all the data access methods are on the `Greeting` entity, rather than on a separate data access class,
+ this is an example of the active record pattern.
+
+The `Greeting` table will have one column, a field called `name`.
+
+=== Writing data
+
+To use the new entity, update the `hello` method to start writing some data.
+
+Change the method to the following:
+
+[source, java]
+----
+@GET
+@Transactional
+@Produces(MediaType.TEXT_PLAIN)
+public String hello(@QueryParam("name") String name) {
+   Greeting greeting = new Greeting();
+   greeting.name = name;
+   greeting.persist();
+   return "Hello " + name;
+}
+----
+
+Don't forget the `@Transactional` method, which ensures writes are wrapped
+in a transaction.
+
+Generally, you shouldn't do state updates on a `GET` REST method, but here it makes
+trying things out simpler. Let's assume what's being written is a logging "side effect",
+rather than a meaningful state changes!
+
+Try out the updated endpoint by visiting http://localhost:8080/hello?name=Bloom.
+You should see a "Hello Bloom" message.
+
+=== Reading data
+Although the new persistence code seems to be working without errors, how
+do you know anything is being written to the database?
+
+Add a second REST method to `GreetingResource`.
+
+[source, java]
+----
+@GET
+@Path("names")
+@Produces(MediaType.TEXT_PLAIN)
+public String names() {
+    List<Greeting> greetings = Greeting.listAll();
+       String names = greetings.stream().map(g-> g.name)
+       .collect(Collectors.joining (", "));
+       return "I've said hello to " + names;
+}
+----
+
+To try it out, visit http://localhost:8080/hello?name=Bloom, and then http://localhost/hello/names.
+
+You should see the following message: "I've said hello to Bloom".
+
+[IMPORTANT]
```

Comment:
Is `IMPORTANT` a thing in asciidoc? I remember using `CAUTION` but maybe I missed `IMPORTANT`.

Reply:
I wasn't sure what kind of admonition to use, so I did a search. We seem to have lots of `IMPORTANT`. I don't know if it's an official part of the spec of just something we added on.

<img width="1511" alt="image" src="https://github.com/quarkusio/quarkus/assets/11509290/dcec2bf7-3912-4040-8777-d12c389c5890">